index.js

```diff
@@ -888,6 +890,9 @@ function buildValue (context, location, input) {
         ${index === 0 ? 'if' : 'else if'}(validator.validate("${schemaRef}", ${input}))
           ${nestedResult}
       `
+      if (schema.type === 'object') {
```

Comment:
can you explain this part please?

Reply:
all this logic is only required when an anyOf/oneOf was within an object. If you look at #630, you can see the logic that was introduced for these kinds of objects. In that case they braces are defined above, but then any of the subsequent entries of the anyOf/oneOf will need the comma added. In advance we dont know which one, so it needs to be added as a prefix to the anyOf-entry producing function when and if it executes.